playbooks/roles/githubsshkeys/tasks/main.yml

```diff
@@ -0,0 +1,27 @@
+- name: install httplib2
+  sudo: true
```

Comment:
`sudo` is deprecated. Our playbooks use [`become`](https://docs.ansible.com/ansible/become.html) to run tasks with sudo anyway, so this line can be removed.